CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 ## Unreleased
 
+## v4.0.1
+
+Fixes loading issues for apps not using the Rails engine.
+
 ## v4.0.0
 
 Stops loading the Rails engine automatically. If you are using the engine, you need to require it explicitly by adding `require 'arturo/engine'` to `application.rb`.
```

Comment:
Do we need to remove/change this line in #144 or add a change to the v4.0.1 release notes?